src/SFA.DAS.EmployerAccounts.Web/Extensions/HtmlHelperExtensions.cs

```diff
@@ -1,44 +0,0 @@
-﻿using SFA.DAS.Authorization;
-using System;
-using System.Linq;
-using System.Web;
-using System.Web.Mvc;
-
-namespace SFA.DAS.EmployerAccounts.Web.Extensions
-{
-    public static class HtmlHelperExtensions
-    {
-        public static MvcHtmlString CommaSeperatedAddressToHtml(this HtmlHelper htmlHelper, string commaSeperatedAddress)
-        {
-            var htmlAddress = commaSeperatedAddress.Split(new[] { ',' }, StringSplitOptions.RemoveEmptyEntries)
-                .Select(line => $"{line.Trim()}<br/>")
-                .Aggregate("", (x, y) => x + y);
-
-            return new MvcHtmlString(htmlAddress);
-        }
-
-        public static AuthorizationResult GetAuthorizationResult(this HtmlHelper htmlHelper, FeatureType featureType)
-        {
-            var authorisationService = DependencyResolver.Current.GetService<IAuthorizationService>();
-            var authorizationResult = authorisationService.GetAuthorizationResult(featureType);
-
-            return authorizationResult;
-        }
-
-        public static bool ViewExists(this HtmlHelper html, string viewName)
-        {
-            var controllerContext = html.ViewContext.Controller.ControllerContext;
-            var result = ViewEngines.Engines.FindView(controllerContext, viewName, null);
-
-            return result.View != null;
-        }
-
-        public static MvcHtmlString CdnLink(this HtmlHelper html, string folderName, string fileName)
```

Comment:
This method is still used in _Layout_v2.cshtml

Reply:
Restored.